telluric/georaster.py

```diff
@@ -1053,52 +1082,67 @@ def _patch_affine(affine):
             affine = affine * Affine.translation(eps, eps)
         return affine
 
-    def reproject(self, new_width, new_height, dest_affine, dtype=None, dst_crs=None, resampling=Resampling.cubic):
+    def reproject(self, new_width, new_height, dest_affine, dst_crs=None, resampling=Resampling.cubic,
```

Comment:
I thought we had agreed on having a simpler `reproject` method. Now the "simple" functionality is implemented in the standalone function, any reason not to have it here too? I recall some discussion about having a `_reproject` internal method but I lost the context.